examples/runkit.js

```diff
@@ -0,0 +1,33 @@
+// Aphrodite server-side rendering example
+
+// Make a component that generates some styles and returns some HTML.
+function render() {
+    const {StyleSheet, css} = require("aphrodite/no-important");
```

Comment:
:P If a console statement is too long in RunKit, it adds an ellipse, which I thought was annoying, so I went with this to fix that. I'm happy to undo that though!

![image](https://cloud.githubusercontent.com/assets/217900/18730137/90d1e01e-8009-11e6-8001-099c626449fa.png)